spec/features/admin/index_spec.rb

```diff
@@ -55,33 +74,37 @@
   end
 
   it "has a header indicating that the user is on the admin dashboard" do
-    visit "/admin"
+    visit "/admin/dashboard"
     expect(page).to have_content("Admin Dashboard")
   end
 
   it "has a link to the admin merchants index" do
-    visit "/admin"
-    expect(page).to have_link("Admin Merchants", href: "/admin/merchants")
+    visit "/admin/dashboard"
+      expect(page).to have_link("Admin Merchants")
+      click_link "Admin Merchants"
+      expect(current_path).to eq(admin_merchants_path)
   end
 
   it "has a link to the admin invoices index" do
-    visit "/admin"
-    expect(page).to have_link("Admin Invoices", href: "/admin/invoices")
+    visit "/admin/dashboard"
+      expect(page).to have_link("Admin Invoices")
+      click_link "Admin Invoices"
+      expect(current_path).to eq(admin_invoices_path)
   end
 
   it "has a list of the top 5 customers who have conducted the largest number of successful
     transactions" do
-      visit "/admin"
+      visit "/admin/dashboard"
 
-      within("#admin-dashboard") do
+      within("#dashboard-customers") do
```

Comment:
It looks like the presence of the content expected in this `within` block is covered by the more robust `within` block on line 107. We might consider deleting this `within` block to DRY up our spec file.

Reply:
I will combine the tests and delete the extra within block.